### DISCUSSION TITLE
CoPilot @ 1.8.3 no longer provides in-line completions, @1.8.2 does.

### DISCUSSION THREAD

**Discussion by thedracle** [community/community#49234] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

For some reason after upgrading to 1.8.3 I no longer get in-line completions in NeoVim.

Downgrading to 1.8.2 via:
Plug 'github/copilot.vim', { 'commit': 'c7d166ebda265370f38cec374e33f02eeec2f857' }

Solves the issue, and I start getting inline completions again.

**Comment by iamcjwilliams** [comment]
@thedracle  please visit this doc to troubleshoot your issue: https://docs.github.com/en/copilot/troubleshooting-github-copilot

**Reply by thedracle** [reply]
I @iamcjwilliams. I am logged in, and :Copilot status says "Enabled and online," I can get a list of completions using :CoPilot panel, or :CoPilot.

If I pin to the previous versions I get inline completions. At the latest version I do not.

**Comment by ldorigo** [comment]
I don't know if this is the issue of OP, but I also seem to be only getting multiple-line suggestions in neovim (not single-line inline suggestions).
